index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav id="navbar">
+        <div class="container">
+          <a href="#" class="page_logo">My Logo</a>
+          <div class="navbar-nav">
+            <button type="button" class="nav-toggle-btn">
+              <img
+                src="./assets/images/icons/Union@2x.png"
+                width="3rem"
+                alt="menu-bar"
+              />
+            </button>
+          </div>
+        </div>
+      </nav>
+      <section id="headline">
+        <div class="container">
+          <h1 class="headline-title">
+            I’m Mariane <br />
+            Glad to see <br />
+            you!
+          </h1>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <a href="#" class="headline-connect-link"> LET’S CONNECT </a>
+
+          <div class="social-links">
+            <a href="#">
+              <img src="./assets/images/icons/twitter.png" alt="twitter" />
+            </a>
+
+            <a href="#">
+              <img src="./assets/images/icons/linkedin.png" alt="linkedin" />
+            </a>
+
+            <a href="#">
+              <img src="./assets/images/icons/medium.png" alt="medium" />
+            </a>
+
+            <a href="#">
+              <img src="./assets/images/icons/github.png" alt="github" />
+            </a>
+
+            <a href="#">
+              <img src="./assets/images/icons/v.png" alt="v" />
+            </a>
+          </div>
+        </div>
```

Comment:
- [ ] Please it would be better to use a `ul` tag here since these icons form a list so semantically it is better to use it.